src/views/product-tutorials-view/components/product-view-content/components/branded-callout/index.tsx

```diff
@@ -3,14 +3,13 @@ import s from './branded-callout.module.css'
 
 function BrandedCallout({
   heading,
-  headingSlug,
   subheading,
   cta,
   product,
 }: BrandedCalloutProps) {
   return (
     <div>
-      <h2 id={headingSlug}>{heading}</h2>
+      <div>{heading}</div>
```

Comment:
`BrandedCallout` doesn't appear in the table of contents, so it feels more correct to not make this a semantic heading. Note that the styling for `BrandedCallout` is in progress in #375.